warehouse/organizations/models.py

```diff
@@ -165,16 +164,46 @@ class Organization(HasEvents, db.Model):
         onupdate=func.now(),
     )
 
-    # TODO: Determine if cascade applies to any of these relationships
     users = orm.relationship(
         User, secondary=OrganizationRole.__table__, backref="organizations"  # type: ignore # noqa
     )
     projects = orm.relationship(
         "Project", secondary=OrganizationProject.__table__, backref="organizations"  # type: ignore # noqa
     )
 
-    # TODO:
-    #    def __acl__(self):
+    def __acl__(self):
+        session = orm.object_session(self)
+
+        # TODO: The "group:admins" and "group:moderators" principals
+        #           automatically get granted permissions in projects
+        #       Should do something similar in organizations?
+
+        # acls = [
+        #     (Allow, "group:admins", "manage:organization"),
+        #     (Allow, "group:moderators", "manage:team"),
+        # ]
```

Comment:
Yes. But as is we currently perform all admin actions via the admin app/interface.

Granting permissions this way would allow admins to use the user-facing interface to make these changes, I assume? That'd be a departure.... @di thoughts here? My preference is probably to maintain the current separation of concerns via the utility sets a la `warehouse/utils/project.py`

Reply:
I've been thinking about doing this vs. maintaining a very similar but separate set of views. I think it's probably better to do the latter, because we can add additional context, and also easily do things like not send emails/notifications/events when admins perform actions

Reply:
Agreed. In a quick review it appears that we do the ACL dance on Projects to set the level of access that PyPI Admins/Moderators have. So we should do similar here, but the use-case won't be a "sudo" mode, just setting the ACLs so we can act on them in warehouse/admin views.